### PR TITLE
Fix for Smoke Test failures

### PR DESCRIPTION
**Pull request by gokultw**
Payment lite smoke tests were failing due to browser screen size & not Logout of crater.
Fixes added.

https://bahmni.atlassian.net/browse/BAH-1554
[E2E Testcase] - Billing in crater with manual creation of Patient in crater

### CODE REVIEW

File: tests/util/browserOperations.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const taikoHelper = require("../util/taikoHelper")
 const headless = process.env.headless_chrome.toLowerCase() === 'true';
 
 beforeSuite(async () => {
-    await openBrowser({headless:headless, args:["--no-sandbox","--start-maximized","--disable-dev-shm-usage","--start-fullscreen",'--use-fake-ui-for-media-stream']})
+    await openBrowser({headless:headless, args:["--no-sandbox","--disable-dev-shm-usage",'--use-fake-ui-for-media-stream',"--window-size=1440,900"]})
```
Comment (gsluthra) [inline tests/util/browserOperations.js:16]: Can we run tests for tablet resolutions like this too? If so, we could have different modes to run all E2E tests in Browser mode and then in tablet mode?

Comment (gokultw) [reply tests/util/browserOperations.js:16]: yes, we can run it with small resolution but our e2e scripts should be updated with updated selectors in most of the places, Like if we have selectors like toLeftOf or toRightOf then in lower resolution we may need to use below or above selectors.

Comment (gsluthra) [reply tests/util/browserOperations.js:16]: Can we do this for Lab-Lite, and see how much effort it is? You can plan this post MVP also. Then we can see if volunteer QAs can do this for other scenarios.

**Review by gsluthra — APPROVED** [review]

File: tests/util/browserOperations.js (hunk 1 above)
Comment (gokultw) [reply tests/util/browserOperations.js:16]: Sure, Will Create a card in Lab Lite.

Comment (gokultw) [reply tests/util/browserOperations.js:16]: https://bahmni.atlassian.net/browse/BAH-2058

**Pull request merged**